File: Service/Implement/ThongBaoService.cs
namespace Service.Implement
{
    public class ThongBaoService : BaseService<ThongBao, IThongBaoRepository>
    , IThongBaoService
    {
    private readonly IThongBaoRepository _ThongBaoRepository;
    public ThongBaoService(IThongBaoRepository ThongBaoRepository) : base(ThongBaoRepository)
    {
    _ThongBaoRepository = ThongBaoRepository;
    }
    }
    }

File: Map/src/app/shared/ThongBao.service.ts
﻿import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThongBao } from 'src/app/shared/ThongBao.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThongBaoService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThongBao";
    }
}

